internal/broker/export_test.go

```diff
@@ -19,6 +19,10 @@ func (cfg *Config) SetHomeBaseDir(homeBaseDir string) {
 	cfg.homeBaseDir = homeBaseDir
 }
 
+func (cfg *Config) SetAllowedUsers(allowedUsers map[string]bool) {
```

Comment:
It looks like these methods are added to the `Config` struct that is defined and used in `broker.go`, is this intentional?

Reply:
Yes, that is intentional. These methods are defined in `export_test.go` instead of `broker.go` because they should only be used in tests.